packages/core/src/lib/table/table.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, it, expect } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import Table from './table.svelte';
+import TableWith3Cols from './table.spec.svelte';
+
+describe('Table', () => {
+  it('Renders default variant', () => {
+    render(Table);
+    expect(screen.getByRole('table')).toBeVisible();
+    expect(screen.getByRole('table')).toHaveClass('table-auto');
+  });
+
+  it('Renders auto variant', () => {
+    render(Table, { variant: 'auto' });
+    expect(screen.getByRole('table')).toBeVisible();
+    expect(screen.getByRole('table')).toHaveClass('table-auto');
+  });
+
+  it('Renders fixed variant', () => {
+    render(Table, { variant: 'fixed' });
+    expect(screen.getByRole('table')).toBeVisible();
+    expect(screen.getByRole('table')).toHaveClass('table-fixed');
+  });
+
+  it('Renders columns with widths set by cols attribute', () => {
+    const cols = ['20%', '30%', '50%'];
+    const { container } = render(TableWith3Cols, { cols });
+
+    for (const [i, width] of cols.entries()) {
```

Comment:
this is recommended by the linter but using `cols.entries()` is firing a different lint error about JS versions

Reply:
Interesting! What's the error? Let me know if you want to take a look together.

Reply:
the error is:
```
Type 'IterableIterator<[number, string]>' can only be iterated through when using the '--downlevelIteration' flag or with a '--target' of 'es2015' or higher.
```

not sure which tag we should use to resolve -- what version of JS are we using?